web/pdf_page_view.js

```diff
@@ -582,6 +603,50 @@ var PDFPageView = (function PDFPageViewClosure() {
 
       return result;
     },
+
+    paintOnSvg: function PDFPageView_paintOnSvg(canvasWrapper) {
+      if (typeof PDFJSDev !== 'undefined' &&
+          PDFJSDev.test('FIREFOX || MOZCENTRAL || CHROME')) {
+         return Promise.resolve('SVG rendering is not supported.');
+      }
+
+      var cancelled = false;
+      var ensureNotCancelled = function () {
+        if (cancelled) {
+          throw 'cancelled';
+        }
+      };
+
+      var self = this;
+      var pdfPage = this.pdfPage;
+      var SVGGraphics = pdfjsLib.SVGGraphics;
+      var viewport = this.viewport;
+      var actualSizeViewport = viewport.clone({scale: CSS_UNITS});
+      var promise = pdfPage.getOperatorList().then(function (opList) {
+        ensureNotCancelled();
+        var svgGfx = new SVGGraphics(pdfPage.commonObjs, pdfPage.objs);
+        return svgGfx.getSVG(opList, actualSizeViewport).then(function (svg) {
+          ensureNotCancelled();
+          self.svg = svg;
+          svg._viewport = actualSizeViewport;
```

Comment:
Nothing in practice, but I assume it's there simply to maintain parity with the `canvas` code; see https://github.com/mozilla/pdf.js/blob/master/web/pdf_page_view.js#L403-L404 and https://github.com/mozilla/pdf.js/blob/master/web/pdf_page_view.js#L259-L260.